configure.ac

```diff
@@ -54,7 +54,7 @@ dnl Catch ancient versions of pkg-config below 0.27
 AX_REQUIRE_DEFINED([PKG_INSTALLDIR])
 PKG_INSTALLDIR
 
-LT_INIT
+LT_INIT([disable-static])
```

Comment:
I guess it is better to add info about new behavior to CHANGELOG.md.

Reply:
better now?